packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -2244,7 +2244,8 @@ describe( 'Rich link previews', () => {
 
 		const titlePreview = screen.getByText( selectedLink.title );
 
-		expect( titlePreview ).toHaveClass(
+		// eslint-disable-next-line testing-library/no-node-access
+		expect( titlePreview.parentElement ).toHaveClass(
```

Comment:
The test was failing because now we have an extra span wrapping the title in the preview because we are using `<Truncate>`. I don't know how to do this more cleanly! In all honesty, this same test file does this in another instance in the same way.